pages/api/bib/[id]/items.ts

```diff
@@ -0,0 +1,32 @@
+import type { NextApiRequest, NextApiResponse } from "next"
+import { fetchBib } from "../../../../src/server/api/bib"
+
+/**
+ * Item fetching route handler for Bib page
+ *
+ * Used for pagination, item filtering, and view all.
+ *
+ */
+async function handler(req: NextApiRequest, res: NextApiResponse) {
+  const id = req.query.id as string
+  const { discoveryBibResult, status } = await fetchBib(id, req.query)
+  const items = discoveryBibResult?.items
+
+  if (req.method === "GET") {
+    if (status !== 200 || !items?.length) {
+      res.status(500).json({
```

Comment:
Perhaps 400? 500 is more for an internal server error but this is an API fetch error.

Reply:
Good call, fixed.